lab03-kaylee/README.md

```diff
@@ -0,0 +1,13 @@
+##Lab03 Overview
```

Comment:
Good README! I'm glad you wrote down your roadblocks. Those are definitely common roadblocks everyone has, but I think you'll be on your way to getting it much better down the line.